report/data_sources/hubspot_client.py

```diff
@@ -17,7 +32,16 @@ class HubspotClient:
     class ObjectType:  # pylint: disable=too-few-public-methods
         """Hubspot codes for different entity types."""
 
+        CONTACT = "0-1"
         COMPANY = "0-2"
+        DEAL = "0-3"
+        TICKET = "0-5"
+
+    class AssociationObjectType(Enum):
+        CONTACT = "Contacts"
+        COMPANY = "Companies"
+        DEAL = "Deals"
+        TICKET = "Tickets"
```

Comment:
We don't need all these values, but once I found them, I thought I might as well put them in